### PR TITLE
Basic Homepage in React(Without Functionality)

### PR DESCRIPTION
**Pull request by kirti2602**
**FEAT:-**
1. In this PR, you'll find the basic HTML, CSS e-commerce page code converted into JSX syntax only. 
2. No routing done yet. 
Please feel free to leave an honest review. Thank you

### CODE REVIEW

**Review by Nikhil-Nirmal — APPROVED** [review]
Everything looks good.

**Review by tanishk15sharma — APPROVED** [review]
your code is good to merge

File: src/components/footer/Footer.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+      <div class="footer-links flex">
+        <a href="https://github.com/kirti2602">
+          <i class="fab fa-github"></i>
+        </a>
+        <a href="https://www.linkedin.com/in/kirti--sharma/">
+          <i class="fab fa-linkedin"></i>
+        </a>
+        <a href="https://twitter.com/_keeerti">
+          <i class="fab fa-twitter"></i>
```
Comment (RahullRawat) [inline src/components/footer/Footer.js]: Here you have used class instead of className

File: src/components/navbar/NavBar.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+          miracuStore
+        </a>
+      </h2>
+
+      <div className="nav-links flex">
+        <a href="wishlist/wishlist.html">
+          <i class="fas fa-heart"></i>
+        </a>
+        <a href="cart/cart.html">
+          <i class="fas fa-shopping-cart"></i>
```
Comment (RahullRawat) [inline src/components/navbar/NavBar.js]: Same here

**Review by RahullRawat — APPROVED** [review]

**Pull request merged**